Sources/StreamVideoSwiftUI/Utils/Camera.swift

```diff
@@ -320,3 +322,6 @@ private extension UIScreen {
 
 @available(iOS 14.0, *)
 private let logger = Logger(subsystem: "com.apple.swiftplaygroundscontent.capturingphotos", category: "Camera")
```

Comment:
Are we using this one? Probably worth removing or unifying it to our log system (that also uses Oslog underneath)

Reply:
yes, good catch, removed it and using our log system instead